docs/examples/swift-api-walkthrough/api-walkthrough/ViewController.swift

```diff
@@ -259,10 +259,9 @@ class ViewController: UIViewController {
         }
     }
     
-    // missing the count
-    // see https://developer.couchbase.com/documentation/server/4.6/n1ql/n1ql-language-reference/selectintro.html
     func groupByQuery() {
         let query = Query.select(
+                SelectResult.expression(Function.count("*")),
                 SelectResult.expression(Expression.property("country")),
                 SelectResult.expression(Expression.property("tz"))
             )
```

Comment:
@pasin also updated this example per [this link](https://github.com/couchbase/couchbase-lite-ios/blob/feature/2.0/Swift/Tests/QueryTest.swift#L378). Noticed the n1ql [example](https://developer.couchbase.com/documentation/server/4.6/n1ql/n1ql-language-reference/selectintro.html#topic_xwp_r3k_dx__group-sort-limit) uses `count("*")` and the CBL unit test `count(1)`. Is there any difference in CBL between `count(1)` and `count("*")`?